packages/odata-service-writer/test/index.test.ts

```diff
@@ -73,4 +73,41 @@ describe('ODataService templates', () => {
         const fsEditor = await generate(testDir, validServiceConfig as OdataService, fs);
         expect((fsEditor as any).dump(testDir)).toMatchSnapshot();
     });
+
+    it('generate: project with local annotations', async () => {
+        const serviceConfigWithAnnotations: OdataService = {
+            url: 'http://localhost',
+            path: '/sap/odata/testme',
+            version: OdataVersion.v2,
+            metadata: readFileSync(join(__dirname, 'test-data', 'sepmra_prod_man_v2', `metadata.xml`), 'utf-8'),
```

Comment:
test is async. Should be able to use the async read file too?

Reply:
I would still have to await - which effectively makes it sync again. But I dont think the test should be async at all so I can remove that thx